packages/next-codemod/bin/upgrade.ts

```diff
@@ -200,25 +199,26 @@ async function suggestCodemods(
   initialNextVersion: string,
   targetNextVersion: string
 ): Promise<string[]> {
-  const initialVersionIndex = availableCodemods.findIndex(
+  const initialVersionIndex = TRANSFORMER_INQUIRER_CHOICES.findIndex(
     (versionCodemods) =>
       compareVersions(versionCodemods.version, initialNextVersion) > 0
   )
   if (initialVersionIndex === -1) {
     return []
   }
 
-  let targetVersionIndex = availableCodemods.findIndex(
+  let targetVersionIndex = TRANSFORMER_INQUIRER_CHOICES.findIndex(
     (versionCodemods) =>
       compareVersions(versionCodemods.version, targetNextVersion) > 0
   )
   if (targetVersionIndex === -1) {
-    targetVersionIndex = availableCodemods.length
+    targetVersionIndex = TRANSFORMER_INQUIRER_CHOICES.length
   }
 
-  const relevantCodemods = availableCodemods
-    .slice(initialVersionIndex, targetVersionIndex)
-    .flatMap((versionCodemods) => versionCodemods.codemods)
+  const relevantCodemods = TRANSFORMER_INQUIRER_CHOICES.slice(
```

Comment:
this is a nice change. I feel like it'd also make sense to show the most recent codemods first, so users don't have to scroll all the way from v11 if they are only interested in the latest codemod.

Reply:
reversed!